### PR TITLE
Saved Json to MongoDB, created recipeId as ID, did code review, added Feed Controller, Repository, Services, ServicesImpl

### PR DESCRIPTION
**Pull request by pavelb2p**
1. Saved Json to MongoDB (POST method).
2. Created recipeId as ID in Feed class (recipeId == id in MongoDB)
3. Did code review(RESTful naming)
4. Added classes Feed entity, FeedController, FeedRepository, FeedServices, FeedServicesImpl

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-FROM openjdk:latest
+#FROM openjdk:latest
```
Comment (kastyan-kg) [inline Dockerfile:1]: why all the properties are commented?

File: pom.xml [hunk 2]
```diff
@@ -7,14 +7,18 @@
         <artifactId>spring-boot-starter-parent</artifactId>
         <version>2.6.4</version>
         <relativePath/> <!-- lookup parent from repository -->
+
```
Comment (kastyan-kg) [inline pom.xml:10]: format this file properly - remove all comments, extra spaces, tabulations etc

File: pom.xml [hunk 3]
```diff
@@ -7,14 +7,18 @@
...
     </parent>
+
```
Comment (kastyan-kg) [inline pom.xml:12]: same as above, you can use cntrl+ alt + f in intelijjy

File: pom.xml [hunk 4]
```diff
@@ -7,14 +7,18 @@
...
     <groupId>com.example</groupId>
     <artifactId>MyCrudProject</artifactId>
     <version>0.0.1-SNAPSHOT</version>
     <name>MyCrudProject</name>
     <description>MyCrudProject</description>
     <properties>
         <java.version>1.8</java.version>
+
```
Comment (kastyan-kg) [inline pom.xml:20]: same as above

File: pom.xml [hunk 5]
```diff
@@ -48,26 +52,46 @@
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
 
-        <!-- https://mvnrepository.com/artifact/org.mongodb/mongo-java-driver -->
+<!--        last-->
```
Comment (kastyan-kg) [inline pom.xml:55]: remove comments

**Review by kastyan-kg — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (pavelb2p) [reply pom.xml:12]: thanks, i used ctrl+alt+L in my IDEA config

**Pull request closed without merging**